src/main/java/dbProcs/Getter.java

```diff
@@ -1856,6 +1913,7 @@ public static String getUserClassFromName (String ApplicationRoot, String userNa
 	{
 		log.debug("*** Getter.getUserClass ***");
 		String result = new String();
+		userName=userName.toLowerCase();
```

Comment:
Why is this being added?

Reply:
To comply with OWASP auth recommendations to make usernames non-case-sensitive. Also, we've had this issue in our installs